RiotSwiftUI/Modules/Room/PollTimeline/View/PollTimeline.swift

```diff
@@ -0,0 +1,129 @@
+// File created from SimpleUserProfileExample
+// $ createScreen.sh Room/PollEditForm PollEditForm
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct PollTimeline: View {
+    
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    @Environment(\.theme) private var theme: ThemeSwiftUI
+    
+    // MARK: Public
+    
+    @ObservedObject var viewModel: PollTimelineViewModel.Context
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 16.0) {
+            
+            Text(viewModel.viewState.poll.question)
+                .font(theme.fonts.bodySB)
+            
+            VStack(spacing: 24.0) {
+                ForEach(viewModel.viewState.poll.answerOptions) { answerOption in
+                    Button {
+                        viewModel.send(viewAction: .selectAnswerOptionWithIdentifier(answerOption.id))
+                    } label: {
```

Comment:
For readability, it would be nice to extract either the entire button or just the label as `PollTimelineAnswerButton`/`Label`.

Reply:
Why not both? 😅